src/api-service/__app__/instance_config/__init__.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import azure.functions as func
+from onefuzztypes.enums import ErrorCode
+from onefuzztypes.models import Error
+from onefuzztypes.requests import InstanceConfigUpdate
+
+from ..onefuzzlib.config import InstanceConfig
+from ..onefuzzlib.endpoint_authorization import call_if_user, can_modify_config
+from ..onefuzzlib.events import get_events
+from ..onefuzzlib.request import not_ok, ok, parse_request
+
+
+def get(req: func.HttpRequest) -> func.HttpResponse:
+    return ok(InstanceConfig.fetch())
+
+
+def post(req: func.HttpRequest) -> func.HttpResponse:
+    request = parse_request(InstanceConfigUpdate, req)
+    if isinstance(request, Error):
+        return not_ok(request, context="instance_config_update")
+
+    config = InstanceConfig.fetch()
+
+    if not can_modify_config(req, config):
+        return not_ok(
+            Error(code=ErrorCode.INVALID_PERMISSION, errors=["unauthorized"]),
+            context="instance_config_update",
+        )
+
+    for field in config.__fields__:
```

Comment:
Can we document (and ideally, unit test) this update logic? Perhaps as a method on the `InstanceConfig`?